app/controllers/membership_applications_controller.rb

```diff
@@ -51,6 +51,7 @@ def create
       if new_file_uploaded(params)
         helpers.flash_message(:notice, t('.success'))
         redirect_to root_path
+        send_new_membership_application_notice_to_admins(@membership_application)
```

Comment:
This shouldn't go in this conditional.  We should send the email even if `new_file_uploaded(params)` fails.
Put this after the conditional

Reply:
@weedySeaDragon put this after the conditional.